src/pages-and-resources/flexible_peer_grading/Settings.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+
+import AppSettingsModal from '../app-settings-modal/AppSettingsModal';
+import messages from './messages';
+
+const OpenResponsesSettings = ({ intl, onClose }) => (
```

Comment:
As per our discussion yesterday, this must call the advanced settings endpoint to update `force_on_flexible_peer_openassessments`. I suggest looking at the way the wiki app does this